### ISSUE TITLE
replies to replies don't appear immediately

### ISSUE THREAD

**Issue by lkobetz** [zoot-hq/zoot#224]
- they only appear when you navigate away and come back

**Comment by lkobetz** [comment]
This is because the scope of the state when you add a reply is the reply you're replying to, not the parent reply, so it doesn't rerender the parent reply until the whole page rerenders. We need a way to rerender the parent message, not the currentMessage.

**Comment by malkamoskowicz** [comment]
@lkobetz Have you looked into the child_changed method form firebase?

**Comment by malkamoskowicz** [comment]
Take a look at how we use the getUpdatedNumOnline function in Fire.js and ChatList.js, it should be similar functionality.

**Comment by lkobetz** [comment]
@malkamoskowicz Emma and I figured this out yesterday- we added a method to the AllReplies component that adds the new reply to the array on the state, then bound it to the context of AllReplies, passed it to SlackBubble, and called it on the new reply when it's added.
We initially thought about adding a listener like child_added or child_changed, but the main issue was that we needed to force the react component to rerender. Resetting the state of the AllReplies component is what did it. Thanks for the suggestion though!
